components/forms/FieldUploadImage.inc.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * @file controllers/form/FieldUploadImage.inc.php
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2000-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class FieldUploadImage
+ * @ingroup classes_controllers_form
+ *
+ * @brief A field for uploading a file.
+ */
+import ('lib.pkp.components.forms.FieldUpload');
+class FieldUploadImage extends FieldUpload {
```

Comment:
Does this permit multilingual images, e.g. for banners with text on them?

Reply:
It supports `altText` for each image. :+1:

Reply:
OK, but can the image itself be uploaded for each language? This used to be something we supported in OJS 2.x and never came up with a good UI pattern/toolset for doing it in OJS 3.x. It's something we could continue to not support, but I suspect we'll get requests for it.

Reply:
Yep. Every `Field` can be `isMultilingual`, in which case multiple instances will be created on-demand in the browser, and the values merged into a locale object by the `Form`. :tada:

Reply:
Woo! That's something we spent a lot of time talking about but never solved.